src/components/StudentGroup.tsx

```diff
@@ -19,40 +19,45 @@ const ButtonDiv = styled.div`
   display: flex;
 `
 
-type StudentGroupProps = {
+interface IStudentGroup {
```

Comment:
It looks like you've declared this interface twice in two separate files.  If you plan on sharing an interface you can extract it to it's own `interfaces and types` folder and export it.

Reply:
That makes sense.  will do